src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,75 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> list;
```

Comment:
Maybe a better name for this might be tasks? Cos currently it isn't really intuitive what the list will contain.